### DISCUSSION TITLE
I made a site for privacy-focussed SaaS tools and featured Appwrite ✨

### DISCUSSION THREAD

**Discussion by fredrivett** [appwrite/appwrite#3784] [Show and tell]
Hey folks,

Hope it's ok to post here, noticed the `🙌 Show and tell` category and thought this would be a decent spot.

A couple months ago I was looking for **the best privacy-focussed products to build and grow my startup**, and I couldn't find a great place that was aimed at startups, so I built one myself:

📝 [PrivacyShortlist.com](https://privacyshortlist.com/)

You can filter by features, browse by category, see who has the most github stars and recommend the products you love.

❤️ Feel free to give some love to Appwrite here: [Appwrite | Privacy-Focussed 💾 Database](https://privacyshortlist.com/database/product/appwrite)

Would love any honest thoughts and feedback too.

Cheers!
Fred.

**Comment by olivia2801** [comment]
Hey, this looks awesome—love the clean concept behind PrivacyShortlist!

If you're ever curating best SaaS tools for workforce management or productivity tracking, definitely check out Workstatus. It’s great for startups managing remote or hybrid teams—time tracking, productivity insights, and GPS-based attendance, all in one place. Super helpful for staying lean and efficient. 👌

Wishing you all the best with the project!
